egs/tedlium/s5/local/chain/run_tdnn.sh

```diff
@@ -67,35 +65,25 @@ EOF
 fi
 
 # The iVector-extraction and feature-dumping parts are the same as the standard
-# nnet3 setup, and you can skip them by setting "--stage 8" if you have already
+# nnet3 setup, and you can skip them by setting "--stage 9" if you have already
 # run those things.
 
-suffix=
-if [ "$speed_perturb" == "true" ]; then
-  suffix=_sp
-fi
-
-dir=${dir}$suffix
-train_set=train$suffix
-gmm=tri3
-gmm_dir=exp/$gmm
-ali_dir=exp/${gmm}_ali$suffix
+train_set=train
+gmm_dir=exp/tri3
+ali_dir=exp/tri3_ali
 lats_dir=${ali_dir/ali/lats}
-treedir=exp/chain/tri3_tree$suffix
+treedir=exp/chain/tri3_tree
 lang=data/lang_chain
 
 mkdir -p $dir
-# if we are using the speed-perturbed data we need to generate
-# alignments for it.
 local/nnet3/run_ivector_common.sh --stage $stage \
-  --speed-perturb $speed_perturb \
-  --generate-alignments $speed_perturb || exit 1;
+  --speed-perturb true || exit 1;
```

Comment:
I was generating alignments because the baseline cross entropy nnet3 model uses the generated alignments anyway. I'll make a comment that it's optional.